lib/ClimaCoreVTK/test/runtests.jl

```diff
@@ -54,6 +56,7 @@ dir = mktempdir()
         U[t] = u
     end
     writevtk(joinpath(dir, "sphere_vector_series"), times, (U = U,))
+    @test isfile(joinpath(dir, "sphere_vector_series.pvd"))
```

Comment:
can also check that the individual `.vtu` files also exist

Reply:
True. I thought this way was more direct and did not require writing for loops for each test. And I thought that the `.pvd` is created at the end of the collection, so the collection itself was successfully created if the pvd file exists